core/src/main/java/oracle/weblogic/deploy/util/WLSDeployArchive.java

```diff
@@ -615,6 +615,15 @@ public String addApplication(String appPath) throws WLSDeployArchiveIOException
         return newName;
     }
 
+    public String readdApplication(String appPath, String tempFile) throws WLSDeployArchiveIOException {
```

Comment:
Is there a typo in the function name?  Two `d`s  or  should it be named something else